corehq/apps/domain/views/base.py

```diff
@@ -17,23 +17,31 @@
 from corehq.apps.users.models import SQLInvitation
 
 
+def covid19(request):
+    return select(request, next_view="app_exchange")
+
 # Domain not required here - we could be selecting it for the first time. See notes domain.decorators
 # about why we need this custom login_required decorator
 @login_required
-def select(request, domain_select_template='domain/select.html', do_not_redirect=False):
+def select(request, do_not_redirect=False, next_view=None):
     domains_for_user = Domain.active_for_user(request.user)
     if not domains_for_user:
         return redirect('registration_domain')
 
     email = request.couch_user.get_email()
     open_invitations = [e for e in SQLInvitation.by_email(email) if not e.is_expired]
 
+    # next_view must be a url that expects exactly one parameter, a domain name
+    next_view = next_view or request.GET.get('next_view')
     additional_context = {
         'domains_for_user': domains_for_user,
-        'open_invitations': open_invitations,
+        'open_invitations': [] if bool(next_view) else open_invitations,
```

Comment:
nit
```suggestion
        'open_invitations': [] if next_view else open_invitations,
```

Reply:
I like the `bool` because it's explicit, although perhaps being inside an `if...else` is explicit enough.